infrastructure/cli/pkg/kube/context.go

```diff
@@ -21,10 +20,6 @@ func New(kubeConfigPath, contextName string) KubeCtx {
 
 func (c *KubeCtx) GetClientSet() (*kubernetes.Clientset, error) {
 	if c.ContextName == "" {
-		if c.KubeConfigPath == "" {
```

Comment:
Why the removal? 🤔

Reply:
I agree that the nested `if` condition doesn't look great, but if we let this pass `BuildConfigFromFlags` will return the current context which means that it could talk to whatever current profile the user is using.

Reply:
I actually can't imagine why. Do you know something about it @ljupcovangelski ?

Reply:
I'm not sure, I don't recall changing anything in this file. Will bring it back.